dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/DefaultEnrollmentService.java

```diff
@@ -101,7 +102,7 @@ public Enrollment getEnrollment(
       throws ForbiddenException {
     User user = currentUserService.getCurrentUser();
     List<String> errors = trackerAccessManager.canRead(user, enrollment, false);
-    if (!errors.isEmpty()) {
+    if (!errors.isEmpty() && !user.isAuthorized(F_TRACKED_ENTITY_INSTANCE_SEARCH_IN_ALL_ORGUNITS)) {
```

Comment:
I am a little bit confused.
If user has the authority but he is not using ALL mode, should we still skip the checks

Reply:
nice catch, no we should skip the validation in this case